usaspending_api/broker/models.py

```diff
@@ -22,3 +22,18 @@ class Meta:
         managed = True
         unique_together = (("last_load_date", "external_data_type"),)
         db_table = "external_data_load_date"
+
+
+class DeltaTableLoadVersion(models.Model):
+    delta_table_load_version_id = models.AutoField(primary_key=True)
+    name = models.TextField(blank=False, null=False)
+    last_version_copied_to_staging = models.IntegerField(default=-1, null=False)
+    last_version_copied_to_live = models.IntegerField(default=-1, null=False)
+    description = models.TextField(blank=True, null=True)
+    create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    update_date = models.DateTimeField(auto_now=True, null=True)
+
+    class Meta:
+        managed = True
+        # Schema is set to `public` in migration `0003_deltatableloadversion`
```

Comment:
Is this comment really necessary? I feel like it sets a precedent, that is anytime something changes in this model we need to add a comment in this class making known which migrations makes a change.